controllers/login.controller.js

```diff
@@ -0,0 +1,18 @@
+const { validationResult } = require("express-validator");
+
+function LoginController() {
+  this.store = function store(req, res, next) {
+    const errors = validationResult(req);
+
+    if (!errors.isEmpty())
```

Comment:
The best practice is to encode `if` statements using curly braces around imperative statements. This isn't an absolute rule, but it does tend to prevent unanticipated side effects resulting from adding new statements to the if-block and then forgetting to add the curly braces. It's also a visual indicator that makes it clear which set of statements are executive if the conditional evaluates to true.